scripts/pre_commit_linter.py

```diff
@@ -463,8 +470,239 @@ def _redirect_stdout(new_target):
         sys.stdout = old_target
 
 
+def _get_all_files():
+    """This function is used to get the list of all files for linting and
+    pattern checks.
+
+    Returns:
+        list(str, bool). The list of files for linting and pattern checks and
+            bool for mode.
+    """
+    eslintignore_path = os.path.join(os.getcwd(), '.eslintignore')
+    parsed_args = _PARSER.parse_args()
+    # Default mode is non-verbose mode, if arguments contains -v flag it will
+    # be made True, which will represent verbose mode.
+    mode = False
```

Comment:
`mode = bool(parsed_args.v)` and drop the next two lines.

You know, I would suggest using --verbose instead for the arg, btw. It would be easier to understand in this code. "v" is often used for "version".

Reply:
Done